libexec/tfenv-help

```diff
@@ -13,6 +13,6 @@ Commands:
    version-name  Print current version
    init          Update environment to use 'tfenv' correctly.
```

Comment:
Single quotes were changed to doubule quotes to make `tfenv` encased by single quotes in the help output, which seemed to be the intended behavior.